be/src/storage/task/engine_alter_tablet_task.cpp

```diff
@@ -39,18 +39,22 @@ EngineAlterTabletTask::EngineAlterTabletTask(const TAlterTabletReqV2& request, i
 OLAPStatus EngineAlterTabletTask::execute() {
     StarRocksMetrics::instance()->create_rollup_requests_total.increment(1);
 
-    SchemaChangeHandler handler;
-    OLAPStatus res = handler.process_alter_tablet_v2(_alter_tablet_req);
+    vectorized::SchemaChangeHandler handler;
```

Comment:
It is better to use a configuration item to decide whether to turn on vectorised SC

Reply:
> It is better to use a configuration item to decide whether to turn on vectorised SC

done,https://github.com/StarRocks/starrocks/pull/453/commits/ef5f9723097a36c2a7c53db0f51d3fe35901dc92,PTAL